android/app/src/androidTest/java/org/fossasia/openevent/APITest.java

```diff
@@ -76,7 +76,7 @@ public void onFailure(Call<List<Sponsor>> call, Throwable t) {
 
     public void testEventAPIResponse() throws Exception {
```

Comment:
No, unit tests should not access network at all. It becomes an instrumentation test or E2E test then